### DISCUSSION TITLE
Treasure Island V2 / Pirate Zone

### DISCUSSION THREAD

**Discussion by Blasta-alasta** [sunflower-land/sunflower-land#3659] [Ideas]
Have some ides for for the revamp of Treasure Island (excuse bad grammar)...

**IDEA**
- Create an Island similar to the Plaza/Beach (could be the Port where the Pirate Zone is), with 4 distinct areas/zones that have 25-30 spots to dig in.
- One extra area on the island (The Pirate Camp) that has a Shop and a Market Place with NPCs to interact with.
- Remove ability to ‘cash out’ current Bounty into SFL, and add one-time use items that can be crafted at the market place.
- Increases the amount of Treasure Bounty to 30+ items. 
- Change the digging odds, in so that every dig guarantees an prize. Pack out these odds with different food as was done with first iteration of TI.
- Create incentives for ongoing engagement in the feature through codex, quest and desirable items at the Market Place.
- Create a new seasonal function that aims to widen the audience, releasing time-limited boost bearing items that can be crafted with Bounty.

**GOALS**
- Continuous Engagement
- Complete Resource Loop
- Not completely RNG reliant, some strategy involved
- Compliments other features (fishing, flowers, mutant crops, etc)
- Incorporates community suggestions around fairness to all players
- Ensure this new iteration complies with any grant or legal conditions.
- Encourages Community Strategic Involvement
- Rewards current Bounty hoarders, but doesn’t punish those without a stash

**DIGGING**
The basic idea of digging is the same as Treasure Island V1; you use Sand Shovels or Drills to dig for Bounty and other Items. There are some changes though:

Every Dig guarantees a Prize

- There are far more Bounties and other goodies to be found
- There are different Areas that a player can dig, each with differing odds
- There are options for addition crop burning to take advantage of all the digging areas

The same amount of Shovels (25) and Drills (5) can be crafted per Blockbuck, but it may be cool to implement the shovel rack idea posed by the community last year (that would give extra % of tools similar to the toolshed) as something that is craftable at Goldtooth’s Store all year round, and as something that players that enjoy digging a lot can work towards.

**AREAS:**

The 4 different areas will have different drops rates for certain treasures. (Further idea - these odds could also be switched up a bit depending on the Dawnlight and Dusktide conditions)/

Players will have to pay a daily offering to access the different zones. These fees would be nominal and paid in resources, mainly crops.

_Area Entry Fee Example:_
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/55673bb0-76ac-423d-880a-7e9efdc1b090)
Allowing 25 daily digs per area (eg 25 spots to possibly dig in each area), players will have to choose to make a decision which is the best area to visit and whether they can afford to visit multiple areas in one day. As the conditions change this will encourage the community to share their data and findings in order to meet the differing goals the island has to offer.

**TREASURE BOUNTY & Other Items**

Add a Bounty Tab to the Codex and have 30+ Bounties of differing drop rates to be found to compete it.

Apart from Bounty, digging can also reward players with foods and other items.

Here are some ideas for a 30+ Bounties (including the current ones). For the game-loop to be complete the Bounty that we already have leftover from V1 have all had the SFL sell values removed. This removes Players ability to just ‘cash out’ with their treasure. Instead all the Treasure could be given different utility; that they can be used mainly to craft ‘temporary boosts’ (similar to the Fertilisers players currently get from composing) and other one-use burnable items (such as tools).

The Bounty is split into Common, Uncommon, Rare, and Ultra Rare (or whatever) similar to the Fish and Flowers in the Codex. This allows for Treasure Hunting to be integrated into the Codex. From here the old milestone rewards (for the basic wearables, pirate potion etc) can added as well as some newer harder to obtain ones.

Since the certain Bounties can only be found in certain areas under certain conditions, the codex can be used also to track where each item has been found (similar to how the flower discovery works).

![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/8f6e8303-e047-4239-8e68-4615fcc1aa59)

**THE PIRATE MARKET (NPCS)**
In the Port Market Bounty can be sorted in four different crafting groups, with some crossover between them. These crafting groups related to the places that they can used at; “Witches Items”, “Cooks Items”, “Seasonal Crafting Items”, and “Recyclables”. This is not so important but helps to understand the logic of the different items craftable at the Pirate Market.

The Pirate Market area comprises of 1 delivery requiring NPC, and 4 shops or stalls:

**- _Delivery NPC ‘Buccaneer Billy’:_** Requires Bounty Items (only) in exchange for SFL. Here the Treasure Map (SFT) boosts can take affect.

The four shops are where players can spend their Bounty, each corresponding to the different Bounty groups.

**- _The Witches Stall :_** here players buy (craft) Strange & Special Items in exchange for certain Bounty. Alongside some Flowers and Fish, Bounty that can be used here: Sea Cucumbers, Seaweed, Barnacles, Starfish, Coral, Whale Tooth, Pyrite Fossil, etc. The items here are one-time use, similar to fertilisers, with a suggestion to make Magic Beans also craftable for some rarer bounties.
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/14dc3a59-8738-47cb-b47e-59fb228bf666)

**- _Cookie’s (The Pirate Chef) Crab Shack :_** here players (instantly) craft high experience foods in exchange for certain Bounty. Bounty (Fish and =that can be used here: Crabs, Black Crabs, Seaweed, Cockles, Pipis, Clams, Scallops, etc. Other Foods, Crops, Fruits and Fish may be required to craft some of these Foods.
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/f7d4a819-3b2e-40af-868a-c6147d08838d)

**- _Pirate Scrap Yard :_** here players can exchange recyclable Bounty for tools current and new, and other special like items that can improve their resource gathering abilities
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/a5d49cd6-698f-422a-98eb-81cbc372c7d3)

**- _Goldtooth’s store :_** a few things going on here, but basically here you buy your tools for digging and craft collectables, some seasonal (time limited) and some that can be crafted by all players that meet the crafting requirements:

1. Buy your Tools (shovels and drills) for digging.
2. Craft seasonal collectables - (see below)
3. Open Daily Pirate Chest
4. Obtain the first clue as to how to complete the Quest:

**GOLDTOOTH’S QUEST**

Ongoing quest that requires a long time to complete. With a substantial prize at the end. My idea is that the first 100 people that reach this goal will be get a better reward than everyone else who could eventually complete the quest as well, and maybe a second teir prize for the second 200 or whatever. 

The idea is that Players will have to gather pieces of a old map. The map has been torn into 7 pieces, 6 of which are lost. The sixth is in Goldtooth’s hands and he will only let you have if you bring him the other 6 missing pieces.

***Seasonal Collectables:**

My suggestion for this store’s craftables are Seasonal Collectables but with a different approach.; using a popular idea I’ve heard floated in the community in regards to the expiry of Seasonal Tickets (like feathers, tulip etc):

Many ‘younger’ players have talked of why they cannot accumulate seasonal tickets and use them in other seasons to gain an edge on more higher tiered players that used their tickets in previous seasons. Personally I like that each season we start a fresh, but also think this idea could be employed here, and in combination with time (instead of supply) limited items could make for some interesting long term gameplay.

The idea is that every 3 months (in line with the changing seasons) at least 3 new boost bearing items would be added to the store, whilst the previous season’s minting finishes. These 3 items’ boosts should be tiered with the difficulty of obtaining them (good time to note that I love idea that these would not stack with each other if the boosts were similar, making the one desirable and the other two ‘cosmetic’ if a player were to play hardcore and try and score all three… but the object will be to make the drops rates of Bounty used high enough that these items are still semi-scarce). 
The twist here is when the season and minting period ends, the main Bounty Requirements (compasses) do not become obsolete, and are carried over to the new seasons items. 

Meaning if a player was very close to crafting ‘**Item 3**’ but the minting time closed, they would have significant advantage in minting ‘**Item C**’ from the next season’s release. Because these items are time limited, players can continue racing for the require Bounty all season long, as collecting extra will give advantages for upcoming drops. 

Utilising time limited items also greatly reduces any significant advantage current players that are holding any old TI Bounty have - they still have an advantage, but not one that means someone newer or without current Bounty, would miss out because of.

Basic Example of how this could work:
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/fb44093a-960e-4f53-b0ff-856d1ed1fb8b)
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/09efb147-c089-44ca-be02-1cebdcb4bafb)
![image](https://github.com/sunflower-land/sunflower-land/assets/102267293/bf9d189e-3180-4763-9915-b389e7b5caeb)

**HOW GOALS ARE ACTIONED:**

- Continuous Engagement: through the seasonal collectables, the temporary boosts, tools, high experience foods, I believe this portal offers many reasons to for continuous engagement.

- Complete Resource Loop: Every bounty item can used to craft something else. Players burn wood, stone, iron, gold and SFL on Shovels and Drills, and in return get small bounty that can be used for boosts, tools, and high experience foods.

- Not completely RNG reliant: I hope that having different Areas with different drop rates for Bounty and other Items add a little bit more strategies to the idea.  I also think exploring the idea of being able to use multiple ‘Rare’ Items to craft those that due to RNG you can’t seem to find. Example, what if there was a Travelling Salesman that could come to the Island every now and again and give players an option to Trade 2 Gold Doubloons for 1 Diamond Earring? Anyway, just a thought to ease the stress RNG can put on players by giving multiple paths to achieving the same goal. 

- Compliments other features (fishing, flowers, mutant crops, etc): I think by using some flowers and fish in some of the crafting requirement at the Pirate Market it ties these features together more. I had the idea to add a third fishing spot and tie it into the lore of Treasure Island, but I think I already have quite a bit going on here in this proposal. I also like the idea that Magic Beans could potential be crafted for a high Bounty cost at the Witches Stall. Another suggestion could be to have this as a prize with a very low drop rate while digging on the island. Again this ties these feature together nicely but incorporating aspects of all parts of the game.

- Incorporates community suggestions around fairness to all players: taken feedback about how the current seasons play out. Many younger players feel that everything is just dominated by the top players and they can’t get ahead (on tickets) because as soon as they start to the season ends and the tickets become obsolete. I like the idea that with Compasses used in conjunction with time limited crafting, players can strategise what is achievable for their play style. Players that miss out in one season or—-so get an advantage in the next with disadvantaging other players new and old. 

- Ensure this new iteration complies with any grant or legal conditions: I’ve read the grants and I believe as everything is circular (no cashing out) that this proposal fall within the guidelines. I’m not entirely sure what the rules state exactly, but everything propose here is non-withdrawable, except the seasonal collectable which I assume are within the terms to be allowed.

- Encourages Community Strategic Involvement: being not completely RNG reliant means players can share their finds and locations.

- Rewards current Bounty hoarders, but doesn’t punish those without a stash: time limped items means that players that have a big stash of bounty may be able to very quickly craft some of the seasonal collectables, but this okay because they won’t run out! Soon these players will use their stash and find themselves on equal footing to other players.

**Example how different drop rates could work with different areas:**
![Screenshot 2024-05-03 at 10 40 53 PM](https://github.com/sunflower-land/sunflower-land/assets/102267293/ae40720f-f82a-4026-ae10-63d50f349a6c)

![Screenshot 2024-05-03 at 10 16 48 PM](https://github.com/sunflower-land/sunflower-land/assets/102267293/14b8e2a0-54ff-4a30-98e0-d3d1f438a54e)

![Screenshot 2024-05-03 at 10 17 06 PM](https://github.com/sunflower-land/sunflower-land/assets/102267293/7c8d3486-bec7-483d-bde1-9bc097105195)

**Comment by TheDcol** [comment]
Woah this very indepth, great detail and great effort put in. I will be forwarding it to the devs in case they don't see it.

I think I get it but still want to go through it a couple of times... thank you Blasta

**Reply by Blasta-alasta** [reply]
Thanks man, it started just as a few ideas but kept evolving, might be a couple of ideas the devs like in there though.

**Reply by TheDcol** [reply]
I'm going to link this thread to my 'overall' one and place it in the Pirate Zone section

**Comment by Dionis404** [comment]
That's really cool! 
I really hope the island comes back

**Comment by TheDcol** [comment]
Just some questions when you have time:

> Create an Island similar to the Plaza/Beach (could be the Port where the Pirate Zone is), with 4 distinct areas/zones that have 25-30 spots to dig in.

Why divide into 4 areas? why not just 1 area?

> Remove ability to ‘cash out’ current Bounty into SFL, and add one-time use items that can be crafted at the market place.

Convert into coins?


> Change the digging odds, in so that every dig guarantees an prize. Pack out these odds with different food as was done with first iteration of TI.

Yes, or add 'sand' resource used to craft?

> Create incentives for ongoing engagement in the feature through codex, quest and desirable items at the Market Place.

Include it in seasonal chores and add items requiring ingredients from pirate zone?

> Create a new seasonal function that aims to widen the audience, releasing time-limited boost bearing items that can be crafted with Bounty.

^ or like you mentioned here

**Reply by Blasta-alasta** [reply]
_Why divide into 4 areas? why not just 1 area?_

You could just have the one area as before, but i thought it might be fun to have different areas within the Zone to dig in, with different Bounty and odds (similar to why we have two fishing spots i guess). That way too, its not just always blind digging, but there can be a little strategy thrown in if you are trying to collect certain bounties for certain things to craft. And with small entrance fees to the different areas, encourages small daily burning.

_Convert into coins?_

Ya, originally i wrote this just before the coin announcement, and the goal was to remove ability to ‘cash out’ Bounty into SFL, as this seemed to be one of the problematic things in regards to the local laws/grant conditions. Now we have coins you could leave the ability to cash out Bounty into Coins, although tbh i think it would better to incentivise players to use their bounty in the Pirate Zone Shops.

_Yes, or add 'sand' resource used to craft?_

Yes! This could also be something added to the odds. Mainly, i think removing 'empty digs' should be the goal. As (sorry again) with fishing, every drop guarantees a prize, which I thing is better all round than nothing, even it is smaller things like Pumpkin Soups, Sunflower Crunch, Crabs, Seaweed etc. I also didn't write it, but think now we are in the land of much higher XP earning, it would be okay to bring back cakes. People hardly bake the old cakes, and they aren't really sellable anymore... and 1200 XP sure aint what it use to be!

What would you envision Sand would be used for if it was to be added as resource? Something added to the crafting ingredients for the SHops, or something else entirely?

_Include it in seasonal chores and add items requiring ingredients from pirate zone?_

Of course! You can bring back the Digging Chores etc too. Tbh although you know i'm not a big fan of RNG crop tickets... I think if they served a greater purpose (ie rewarding farm activity - but thats probably a different subject) they would be more appealing. So saying that you could have a small chance a Dig drops a seasonal ticket, similar to a crop. Buy i'm not sure how popular that idea would be lol.

_Create a new seasonal function that aims to widen the audience, releasing time-limited boost bearing items that can be crafted with Bounty._

This is different. I saw at times 'less advanced' players saying that it would be better if the seasonal tickets didn't expire and you could use therm the next season, so in that 'Whales' don't always dominate (ie a small player could save for 3 seasons and mint an OP item) or if they lost the big auction they wouldn't be left with a bunch of 'useless' tickets - which i believe stops many players from attempting to compete.

I don't think the format should change, but you could have an alternative format in the Pirate Zone for the NFTs here.

Using the compasses as 'tickets' basically - that can be used to mint the NFTs (with other bounty items too), but Time Limited like Stella's shop. So if you get enough 'tickets/compasses' before the closing date you can mint the item, but if not, you can use them for the next seasons items. No one misses out because there is no max supply. This also kind puts players that have stored their bounties up until now at an advantage, but not one that disadvantages any newer player.

I believe this will widen the audience, encouraging more players to partake, and promotes continuous engagement (which was an issue for the last iteration i believe).

**Reply by TheDcol** [reply]
> You could just have the one area as before, but i thought it might be fun to have different areas within the Zone to dig in, with different Bounty and odds (similar to why we have two fishing spots i guess). That way too, its not just always blind digging, but there can be a little strategy thrown in if you are trying to collect certain bounties for certain things to craft. And with small entrance fees to the different areas, encourages small daily burning.

Fair enough, i think to start off 2 is a lot more feasible. More could be added down the line.. if so, we could start with 1 area, add another later on etc.

> Ya, originally i wrote this just before the coin announcement, and the goal was to remove ability to ‘cash out’ Bounty into SFL, as this seemed to be one of the problematic things in regards to the local laws/grant conditions. Now we have coins you could leave the ability to cash out Bounty into Coins, although tbh i think it would better to incentivise players to use their bounty in the Pirate Zone Shops.

Okay cool. 

> [Sand] Yes! This could also be something added to the odds. Mainly, i think removing 'empty digs' should be the goal. As (sorry again) with fishing, every drop guarantees a prize, which I thing is better all round than nothing, even it is smaller things like Pumpkin Soups, Sunflower Crunch, Crabs, Seaweed etc. I also didn't write it, but think now we are in the land of much higher XP earning, it would be okay to bring back cakes. People hardly bake the old cakes, and they aren't really sellable anymore... and 1200 XP sure aint what it use to be!

Happy we agree with sand or guaranteed drops, even if sub-par.

> What would you envision Sand would be used for if it was to be added as resource? Something added to the crafting ingredients for the SHops, or something else entirely?

I think to start it could be for deliveries, but would be ncie if it was used to craft items. Not sure what items though..

> Using the compasses as 'tickets' basically - that can be used to mint the NFTs (with other bounty items too), but Time Limited like Stella's shop. So if you get enough 'tickets/compasses' before the closing date you can mint the item, but if not, you can use them for the next seasons items. No one misses out because there is no max supply

At first it sounded like og TI, where play7ers would have to dig at maximum and hope to be blessed with RNG to get enough tickets.
But if they can use it to mint items in subsequent months, it's less punishing

**Comment by raulmonoanivia** [comment]
Really cool and detailed proposal, I really hope that the devs do it this way
